addons/im_livechat/static/src/core/common/chat_bubble_patch.js

```diff
@@ -0,0 +1,16 @@
+import { ChatBubble } from "@mail/core/common/chat_bubble";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(ChatBubble.prototype, {
+    get showImStatus() {
+        if (
+            this.thread?.channel_type === "livechat" &&
+            this.thread.self_member_id &&
+            this.thread.self_member_id.livechat_member_type === "visitor"
```

Comment:
```suggestion
            this.thread.self_member_id?.livechat_member_type === "visitor"
```

Which can be simplified to

```js
return super.showImStatus && this.self_member_type?.livechat_member_type !== "visitor";